chains/evm/executor/executor.go

```diff
@@ -195,9 +218,10 @@ func (e *Executor) executeProposal(proposals []*chains.Proposal, signatureData *
 		GasLimit: gasLimit,
 	})
 	if err != nil {
+		span.SetStatus(codes.Error, err.Error())
 		return nil, err
 	}
-
+	span.AddEvent("Deposit executed", traceapi.WithAttributes(attribute.String("tx.hash", hash.String())))
```

Comment:
WE need tracing inside the substrate extrinsic sender. Hash by itself doesn't mean anything.
I think the most important part is: the extrinsic is being sent, in what block is it, is it finalized and is there a error on execution.

Reply:
Same for EVM.

Reply:
Add traces for tracking Extrinsic execution.
Although we do not have execution tracking for the EVM side of things we just send the tx and that's it.

Reply:
Add traces for monitored Transactor and Monitor in chainbridge-core with PR https://github.com/ChainSafe/chainbridge-core/pull/350